features.md

```diff
@@ -94,6 +95,15 @@ If disabled in a document, then calls to the [`PaymentRequest` constuctor](https
 * The **feature name** for *payment* is "`payment`"
 * The **default allowlist** for *payment* is `'self'`.
 
+### picture-in-picture
+
+The *picture-in-picture* feature controls whether the current document is allowed to use [Picture In Picture](http://wicg.github.io/picture-in-picture).
+
+If disabled in a document, then calls to [`requestPictureInPicture()`](https://wicg.github.io/picture-in-picture/#dom-htmlvideoelement-requestpictureinpicture) and [`exitPictureInPicture()`](https://wicg.github.io/picture-in-picture/#dom-document-exitpictureinpicture) MUST throw a `SecurityError` and [`pictureInPictureEnabled`](https://wicg.github.io/picture-in-picture/#dom-document-pictureinpictureenabled) MUST return `false`.
```

Comment:
FYI, in case of this comes up for the review, `pictureInPictureEnabled` will reflect more than the Feature Policy status: if the system has PIP disabled or the user blocked PIP on the website, this property will return `false`.